packages/web/src/views/pool/pool-incentivize/components/incentivize-pool-history/incentivize-pool-history-box/IncentivizePoolHistoryBox.tsx

```diff
@@ -0,0 +1,242 @@
+import React from "react";
+import { useTranslation, Trans } from "react-i18next";
+import { css } from "@emotion/react";
+
+import { ExtendedPoolStakingModel } from "@models/pool/pool-staking";
+import { capitalize } from "@utils/string-utils";
+import { useGetPoolDetailByPath } from "@query/pools";
+import { getDateUtcToLocal } from "@common/utils/date-util";
+import { toNumberFormat } from "@utils/number-utils";
+import { PoolMapper } from "@models/pool/mapper/pool-mapper";
+import { makeDisplayTokenAmount } from "@utils/token-utils";
+import { GNS_TOKEN } from "@common/values/token-constant";
+
+import { IncentivizePoolHistoryBoxWrapper } from "./IncentivizePoolHistoryBox.styles";
+import Button, { ButtonHierarchy } from "@components/common/button/Button";
+import IconInfo from "@components/common/icons/IconInfo";
+import Tooltip from "@components/common/tooltip/Tooltip";
+import DoubleLogo from "@components/common/double-logo/DoubleLogo";
+import MissingLogo from "@components/common/missing-logo/MissingLogo";
+
+interface IncentivizePoolHistoryBoxProps {
+  stakingData: ExtendedPoolStakingModel;
+  poolPath: string;
+}
+
+const IncentivizePoolHistoryBox = ({
+  stakingData,
+  poolPath,
+}: IncentivizePoolHistoryBoxProps) => {
+  const { t } = useTranslation();
+
+  const { rewardToken } = stakingData;
+
+  const { data: pool = null } = useGetPoolDetailByPath(poolPath, {
+    enabled: !!poolPath,
+  });
+
+  const currentPool = React.useMemo(() => {
+    const temp = pool ? PoolMapper.toPoolSelectItemInfo(pool) : null;
+    return temp;
+  }, [pool]);
+
+  const isSelected = currentPool != null;
+
+  const doubleLogos = React.useMemo(() => {
+    if (!isSelected) {
+      return { left: "", right: "" };
+    }
+    return {
+      left: currentPool.tokenA.logoURI,
+      right: currentPool.tokenB.logoURI,
+      leftSymbol: currentPool.tokenA.symbol,
+      rightSymbol: currentPool.tokenB.symbol,
+    };
+  }, [currentPool, isSelected]);
+
+  const feeRateStr = React.useMemo(() => {
+    return currentPool?.feeRate ? currentPool.feeRate : "-";
+  }, [currentPool]);
+
+  const formatAmount = (amount: string | null) => {
+    if (!amount) return "-";
```

Comment:
Is it “-” even when it's 0?

Reply:
@jinoosss 
Since the parameter is of type string, I think it should work fine even when it is '0'. What do you think?

Reply:
I agree. I think it should be output normally even if it is 0.